test/performance/benchmarks/broker-pubsub/dev.config

```diff
@@ -0,0 +1,61 @@
+### Creating this benchmark:
+# mako create_benchmark test/performance/broker-pubsub/continuous/dev.config
+### Updating this benchmark:
+# mako update_benchmark test/performance/broker-pubsub/continuous/dev.config
+
+project_name: "Knative"
+benchmark_name: "Development - PubSub Broker Latency & Throughput"
+description: "Measure latency and throughput of the broker using various channels."
+benchmark_key: '6592390798245888'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "grantrodgers@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "xiyue@google.com"
+owner_list: "gracegao@google.com"
+owner_list: "nachocano@google.com"
```

Comment:
can you add chen here as well. I think he is in the eventing continuous already...

Reply:
Added.